modules/nf-core/checkv/downloaddatabase/main.nf

```diff
@@ -0,0 +1,35 @@
+process CHECKV_DOWNLOADDATABASE {
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::checkv=1.0.1" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/checkv:1.0.1--pyhdfd78af_0':
+        'quay.io/biocontainers/checkv:1.0.1--pyhdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(fasta)
```

Comment:
I'm not entirely certain about this. Won't it now download a database for each fasta path?